lib/crypto/c_src/api_ng.c

```diff
@@ -820,14 +822,15 @@ ERL_NIF_TERM ng_crypto_one_time(ErlNifEnv* env, int argc, const ERL_NIF_TERM arg
             if (!enif_realloc_binary(&out_data_bin, (size_t)appended_size))
                 {
                     ret = EXCP_ERROR(env, "Can't reallocate final");
-                    goto err;
+                    goto err_realloc;
                 }
             memcpy(out_data_bin.data + first_part_size, final_data_bin.data, (size_t)final_data_bin.size);
         }
 
     /* Exit here */
     ret = enif_make_binary(env,&out_data_bin);
-
+ err_realloc:
+    enif_release_binary(&out_data_bin);
```

Comment:
This will also release the ErlNifBinary that was made read-only in line 831.
If the reallocation of out_data_bin failed in line 822, the ErlNifBinary is still read-only and will be removed by GC.
So this enif_release_binary is not necessary in either case.

Reply:
Looks like I missed that part! Thanks for the review :)